pkg/crc/input/input.go

```diff
@@ -1,17 +1,23 @@
 package input
 
 import (
+	"errors"
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/code-ready/crc/pkg/crc/output"
+	"golang.org/x/crypto/ssh/terminal"
 	survey "gopkg.in/AlecAivazis/survey.v1"
 )
 
 func PromptUserForYesOrNo(message string, force bool) bool {
 	if force {
 		return true
 	}
+	if !terminal.IsTerminal(int(os.Stdin.Fd())) {
+		return false
+	}
```

Comment:
Do we have the equivalent of `yum --assumeyes`/`yum --assumeno`?

Reply:
No we don't have this.